app/views/components/gds/exportsInputText.scala.html

```diff
@@ -0,0 +1,39 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(govukInput: GovukInput)
+
+@(field: Field, labelKey: String, hintKey: Option[String] = None, isPageHeading: Boolean = false, headingClasses: String = "govuk-label--xl", labelClasses: String = "govuk-label--m")(implicit messages: Messages)
```

Comment:
Is it correct to be `"govuk-label--xl"` ? I'd expect `"govuk-label--l"`

Reply:
I don't this that value is being used (without being over-ridden) on the pages I've done, but good spot.  I'll change it to use the gds.Styles constant and then they can all be easily changed back to "xl" later